.github/workflows/e2e-tests.yml

```
@@ -378,6 +378,7 @@ jobs:
       TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
       TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
       TURBO_CACHE: remote:r
+      OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
```

actionlint (1.7.12)

[error] property "openai_api_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; turbo_team: string; turbo_token: string}

(expression)